### DISCUSSION TITLE
page /search/ does not have a modified date

### DISCUSSION THREAD

**Discussion by ghost** [willfaught/paige#84] [Q&A] [answered]
In version 0.90.1, starting the Hugo server shows the below warning:

```
WARN  layouts/_default/list.atom.xml: page /search/ does not have a modified date
```

I have a file `content/search.md` with:

```
+++
title = 'Search'
description = "Search this site."
layout = "paige/search"
+++
```

And in hugo.toml, the following search-related declarations:

```
[[menu.main]]
identifier = "search"
name = "SEARCH"
url = "/search/"
weight = 110

...
...

[outputs]
home = ["atom", "html", "paige-search", "rss"]
section = ["atom", "html", "rss"]
taxonomy = ["atom", "html", "rss"]
term = ["atom", "html", "rss"]
```

What is the warning about and how can I get rid of it ?

**Comment by willfaught** [comment, marked as answer]
The atom output requires a modified date to be well-formed. You can either set the page's date parameter, or set the page's paige.feed.disable parameter to true, or disable the atom output. For example, when the example site had a search page, it had

```toml
[paige.feed]
disable = true
```

in the front matter.

v0.91.1 (just released) changed the warning wording to explain this. Hopefully it will be clearer.
